src/sentry/utils/committers.py

```diff
@@ -229,6 +230,12 @@ def get_event_file_committers(project, event, frame_limit=25):
         {match for match in commit_path_matches for match in commit_path_matches[match]}
     )
 
+    with sentry_sdk.start_span(op="get_event_file_committers") as span:
+        span.set_data("project_id", project.id)
+        span.set_data("group_id", event.group_id)
+        span.set_data("commit_path_matches", commit_path_matches)
```

Comment:
This seems like it could be pretty large, and could get trimmed by the SDK.

Reply:
@NisanthanNanthakumar would clipped data be fine?

Reply:
@markstory @dashed yeah clipped data is fine. I just need some visibility into the function with production data.